docker/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+pip install -r requirements.txt
+pip install -e /fv3gfs-util
+pip install -e /fv3core
+pip install -e /fv3gfs-physics
```

Comment:
Do these need a `-c constraints.txt`?

Reply:
I don't think so because the docker image already installed the requirements with constraints.